renovate.json

```diff
@@ -30,6 +30,7 @@
   ],
   "ignorePaths": [
     "**/docker/**",
+    ".devfile.yaml",
```

Comment:
Just in case this changes, could we also add an exclusions for `devfile.yaml`, `devfile.yml`, and `.devfile.yml` on the root path?

Reply:
Done!